tests/telemetry/test_telemetry.py

```diff
@@ -122,13 +140,14 @@ def setup_streaming_telemetry(duthosts, rand_one_dut_hostname, localhost,  ptfho
     file_exists = ptfhost.stat(path=gnxi_path + "gnmi_cli_py/py_gnmicli.py")
     pytest_assert(file_exists["stat"]["exists"] is True)
 
+
 def skip_201911_and_older(duthost):
     """ Skip the current test if the DUT version is 201911 or older.
     """
     if parse_version(duthost.kernel_version) <= parse_version('4.9.0'):
         pytest.skip("Test not supported for 201911 images. Skipping the test")
```

Comment:
>201911 [](http://example.com/codeflow?start=44&length=6)

What prevent you to test 201911 image? Should we test 201911 image?

Reply:
Please be aware this is not the new change introduced by this PR.

Reply:
I believe some test cases need skip 20191130 image since the corresponding datasets are not available.